redisearch/client.go

```diff
@@ -46,15 +46,57 @@ func NewClientFromPool(pool *redis.Pool, name string) *Client {
 	return ret
 }
 
-// CreateIndex configues the index and creates it on redis
-func (i *Client) CreateIndex(s *Schema) (err error) {
+// GetRediSearchVersion returns RediSearch version by issuing "MODULE LIST" command
+// and iterating through the availabe modules up until "ft" is found as the name property
+func (c *Client) GetRediSearchVersion() (version int64, err error) {
```

Comment:
This will not work on enteprise

Reply:
@MeirShpilraien made `getRediSearchVersion` internal, so that we're only able to use it on testing.